Simple dice roller app development

Kevin-Guillen opened the pull request:
Simple dice roller app.

<img width="211" alt="SimpleDiceRollerScreen" src="https://user-images.githubusercontent.com/84336422/120946105-b03b9b80-c711-11eb-92fa-9245cf0982a3.PNG">


![simple_dice_roll_gif](https://user-images.githubusercontent.com/84336422/121092509-0831db00-c7c2-11eb-9e55-717a544d3c25.gif)

Kevin-Guillen committed:
Simple dice roller app development

marianosalvetti reviewed (approved):
LG

MarcosCGlobant commented on lib/main.dart:
```diff
@@ -1,113 +1,88 @@
 }
 
-class MyApp extends StatelessWidget {
+class Dice extends StatefulWidget {
```
You could use a more representative name for the class

MarcosCGlobant commented on lib/main.dart:
```diff
@@ -1,113 +1,88 @@
-    });
-  }
+class _DiceState extends State<Dice> {
+  int DiceNumber = 1;
```
Always use camelCase for variables

MarcosCGlobant commented on lib/main.dart:
```diff
@@ -1,113 +1,88 @@
+        decoration: BoxDecoration(
+            image: DecorationImage(
+                image: AssetImage("assets/background/dicebackground3.jpg"),
+                fit: BoxFit.fill)),
```
missed trailing commas in line 48 and 49

MarcosCGlobant commented on lib/main.dart:
```diff
@@ -1,113 +1,88 @@
-              style: Theme.of(context).textTheme.headline4,
+            Expanded(
+              child: Padding(
+                padding: const EdgeInsets.all(60.0),
```
missed trailing comma here

MarcosCGlobant replied:
add value to constants file

MarcosCGlobant commented on lib/main.dart:
```diff
@@ -1,113 +1,88 @@
+              child: Padding(
+                padding: const EdgeInsets.all(60.0),
+                child:
+                    Image.asset('assets/dice/dice$DiceNumber.png', width: 300),
```
missed trailing comma here

MarcosCGlobant replied:
add width value to constants file

MarcosCGlobant commented on lib/main.dart:
```diff
@@ -1,113 +1,88 @@
+              ),
             ),
+            Padding(
+              padding: const EdgeInsets.only(bottom: 80.0),
```
missed trailing comma and add value to constants file

MarcosCGlobant reviewed (commented):
Some constants and trailing commas needed

Kevin-Guillen committed:
- Added missing trailing commas
- Added missing constants and deleted colors constants
- Changed the class name of the dice to a more specific one.
- Corrected the variable name DiceNumber to diceNumber

MarcosCGlobant commented on lib/main.dart:
```diff
@@ -1,113 +1,99 @@
+              image: AssetImage(
+                "assets/background/dicebackground3.jpg",
+              ),
+              fit: BoxFit.fill),
```
missed a trailing comma here

MarcosCGlobant commented on lib/utils/constants.dart:
```diff
@@ -0,0 +1,13 @@
+  static const double buttonThemeHeight = 50.0;
+  static const double playButtonFontSize = 25.0;
+  static const double playButtonLetterSpacing = 1.0;
+  static const double ColumnElementsPadding = 60.0;
```
use lowerCamelCase in this constant

MarcosCGlobant reviewed (commented):
Missed a trailing comma and a camelCase in the constans file

Kevin-Guillen committed:
- Added missing trailing comma
- lowerCamelCase in columnElementsPadding constant

MarcosCGlobant reviewed (approved):
LGTM
